Assets/Tile walking mini game/Scripts/TileColorManager.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class TileColorManager : MonoBehaviour
+{
+    [SerializeField] private Color touchColor;
+
+    [SerializeField] private Color untouchedColor;
+
+    [SerializeField] private Color touchedTwiceColor;
+
+    [SerializeField] private Color obstacleColor;
+
+    public void ChangeTileColorBasedOnStatus(Tile tile)
+    {
+        switch(tile.TileStatus)
+        {
+            case TileStatus.untouched:
+            {

```

Comment:
the brackets inside of the case statements are unnecessary

Reply:
Is it just a more compact way to right switch statements or are there performance benefits as well?